tools/load-test/README.md

```diff
@@ -0,0 +1,62 @@
+# Load Test
+
+
+## Overview
+
+The load test verifies the execution of the Horizontal Pod Autoscaling of functions deployed using Kubeless in Kyma. It runs a Golang application which stresses a function by making thousands of HTTP POST requests in a set period of time defined in an environment variable. At the end of the test execution, a notification with the output is sent to a Slack channel.
+
+## Installation
+
+As the load test has been implemented as `helm chart` the script below is the way how the test must be installed.
+
+```bash
+echo "Installing helm chart..."
+helm install --set slackEndpoint="${SLACK_ENDPOINT}" \
+             --set slackClientToken="${SLACK_CLIENT_TOKEN}" \
+             --set slackClientChannelId="${SLACK_CLIENT_CHANNEL_ID}" \
+             --set loadTestExecutionTimeout="$LT_TIMEOUT" \
+             ${LT_FOLDER_CHART} \
+             --namespace=kyma-system \
+             --name=load-test
+```
+
+These are the parameters you must configure to execute the load test:
+
+ | Parameter | Value | Description |
+ |------|---------------|-------------|
+**slackEndpoint** |`-`| A webhook Slack URL.
+**slackClientToken** |`-`|  A token which is a part of the **slackEndpoint**.
+**slackClientChannelId** |`#channelId`| ID of the Slack channel.
+**loadTestExecutionTimeout** |`30`| Time after which the test execution timeout occurs. All the related metrics to be sent to the Slack channel are collected after the timeout.
+
+### Environment Variables
+
+To run the load test either on a cluster or on a local Minikube, you must set the Helm chart installation parameters. However, on a cluster it is advisable to have environment variables set to default values.
```

Comment:
I'd move this section above the table with parameters (and also above the "These are the parameters..." sentence).